client/src/Style.module.css

```diff
@@ -1,10 +1,25 @@
-body {
+html, body {
   margin: 0;
+  padding: 0;
+  min-height: 100%;
```

Comment:
min-height (in percentage) is not supposed to work if the parent/container element doesn't have a defined height. 
how about height:100% and overflow:auto ?